tests/components/demo/test_geo_location.py

```diff
@@ -59,13 +64,14 @@ def test_setup_platform(self):
                     # ignore home zone state
                     continue
                 assert (
-                    abs(state.attributes["latitude"] - self.hass.config.latitude) < 1.0
+                    abs(state.attributes[ATTR_LATITUDE] - self.hass.config.latitude)
+                    < 1.0
                 )
                 assert (
-                    abs(state.attributes["longitude"] - self.hass.config.longitude)
+                    abs(state.attributes[ATTR_LONGITUDE] - self.hass.config.longitude)
                     < 1.0
                 )
-                assert state.attributes["unit_of_measurement"] == LENGTH_KILOMETERS
+                assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == LENGTH_KILOMETERS
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)